packages/flutter_tools/test/general.shard/features_test.dart

```diff
@@ -398,5 +398,24 @@ void main() {
 
       expect(featureFlags.isWindowsUwpEnabled, false);
     });
+
+    for (final Feature feature in <Feature>[
```

Comment:
@christopherfujino can you double check this diff? I had to resolve a conflict with it, but it seemed trivial enough to include.